index.html

```diff
@@ -65,18 +64,27 @@ <h2 class="intext">A place for kids
             <p> expertise in cutting-edge domains.</p>
         </div>
         <div class="secondhead">
-            <h1>2015.10.15(thu)</h1>
+            <h3>2015.10.15(THU- 16(FRI)</h3>
+            <h4>@THE MICHAEL ASONG FOUNDATION CAMEROON, CENTRAL AFRICA</h4>
         </div>
     </section>
 
     <div class="program">
         <h1>Main program</h1>
         <hr class="line" />
         <div class="All-Programs">
-            <div class="grid-item1"><img src="/Images/Icon1.png" alt="Art Presentation"></div>
-            <div class="grid-item1"><img src="/Images/Icon2.png" alt="Conversations"></div>
-            <div class="grid-item1"><img src="/Images/Icon3.png" alt="Networking"></div>
-            <div class="grid-item1"><img src="/Images/Icon4.png" alt="Presentations"></div>
+            <div class="grid-item1"><img src="/Images/Powerpoint.png" alt="Art Presentation">
+                <h3>Seminars </h3>
+                <h6>Frequent Seminars held to improve student </h6>
+            </div>
+            <div class="grid-item1"><img src="/Images/contributions.png" alt="Networking">
+                <h3>Suggestions</h3>
+                <h6>Frequent Seminars held to improve student </h6>
+            </div>
+            <div class="grid-item1"><img src="/Images/guestSpeaker.png" alt="Presentations">
+                <h3>Guest Speaker</h3>
+                <h6>Frequent Seminars held to improve student </h6>
+            </div>
```

Comment:
- [ ] According to the requirements, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: colors and layout, the main program section doesn't look similar to the original design, kindly add more cards and `See The Whole` should be a link, so kindly use `<button>` or `<a>` tags instead of `<div>` 👌💯

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/183268503-4c3a9ac5-c351-40a9-a899-ea37ce0e0224.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/183268517-6abe95ec-2533-43fb-85db-d692f3971127.png" width="500" /> |

- [ ] The design in mobile version should be like this image, so kindly follow the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

<img src="https://user-images.githubusercontent.com/45231276/181266102-bcbcac63-9707-44de-b805-4acf0219c319.PNG" width="500" />